### PR TITLE
Content single styles

### PR DESCRIPTION
**Pull request by conrad-vanl**
## DESCRIPTION

ezgif-1-04981ff5b229

### What does this PR do, or why is it needed?
Updates styles to `UniversalContentItem` and adds a `WeekendContentItem` view within the content single components.

### What design trade-offs/decisions were made?

Just a quick pass, not 100% to @michael-riddering's styles yet.

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apolloschurchapp/src/content-single/MediaControls/index.js [hunk 1]
```diff
@@ -5,95 +5,79 @@ import { View } from 'react-native';
...
 }))(View);
 
+const StyledMediaThumbnail = styled({ marginVertical: 0 })(MediaThumbnail);
+
 class MediaControls extends PureComponent {
   static propTypes = {
     contentId: PropTypes.string,
   };
 
+  renderPlayButton = ({ action, coverImageSources }) => (
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/content-single/MediaControls/index.js]: Nice! I like the way you refactored this.

File: packages/apolloschurchapp/src/ui/FeaturedCardConnected/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/ui/FeaturedCardConnected/index.js]: What do you think about expanding ConnectedCard so it can take a `Component` and maybe a `query` and so we can reuse that instead of making another connected card?

Comment (conrad-vanl) [reply packages/apolloschurchapp/src/ui/FeaturedCardConnected/index.js]: I'm definitely for it. Only problem to solve is prop mapping. The new card components that have been built don't match the content item schema. ContentCard's prop does match our content item schema.

`coverImage` vs `image`
`summary` vs `description`

etc

I guess we might be able to handle this in the query though to some extent??

Comment (conrad-vanl) [reply packages/apolloschurchapp/src/ui/FeaturedCardConnected/index.js]: I'd LOVE Your help in figuring out a solve here. Because I completely don't like just making new connected card components :joy:

Comment (vinnyjth) [reply packages/apolloschurchapp/src/ui/FeaturedCardConnected/index.js]: My gut says "make the props match" 😂 Or we could do something annoying like a function that maps props names....

File: packages/apolloschurchapp/src/content-single/Features/index.js [hunk 3]
```diff
@@ -21,12 +21,22 @@ const Features = ({ contentId }) => {
         if (loading) return null;
 
         const features = get(node, 'features', []);
-        return features.map(({ __typename, ...feature }) => {
-          const Feature = FEATURE_MAP[__typename];
-          return (
-            <Feature key={feature.id} {...feature} contentId={contentId} />
-          );
-        });
+        if (!features || !features.length) return null;
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/content-single/Features/index.js]: Changes to this look good, thanks for expanding the functionality here.

File: packages/apollos-ui-kit/src/theme/ThemeConsumer.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import withTheme from './withTheme';
```
Comment (vinnyjth) [inline packages/apollos-ui-kit/src/theme/ThemeConsumer.js]: Ohhh nice.

File: packages/apolloschurchapp/src/ui/FeaturedCardConnected/index.js (hunk 2 above)
Comment (burkeshartsis) [reply packages/apolloschurchapp/src/ui/FeaturedCardConnected/index.js]: 100% I had no idea I had misnamed them. We either update the new ones or update the old ones. Whichever you think is the easiest. FWIW, I was planning to do the `ContentCard` refactor today so I'll be touching that code anyways. Not a big deal to me either way. Sorry I even created that problem to begin with. I'm 100% in favor of reusing `ConnectedComponents` where  ever possible.

File: packages/apollos-ui-kit/src/theme/ThemeConsumer.js (hunk 4 above)
Comment (conrad-vanl) [reply packages/apollos-ui-kit/src/theme/ThemeConsumer.js]: Eventually I want to migrate `Theme` over to the new context components / refactor `Theme` not to use HOCs at all...but figured this atleast exposed the API

**Comment by vinnyjth** [comment]
@burkeshartsis will you have a chance to review this today? If not, I'll take over it.

**Comment by vinnyjth** [comment]
Once I review and merge #961 I'm going to take another look at this PR and hopefully merge it.

**Comment by conrad-vanl** [comment]
@vinnyjth it appears #961 is merged, any thing I can do on this?

**Comment by vinnyjth** [comment]
@conrad-vanl I am now waiting on #967 so I can remove your custom connected component...

**Comment by vinnyjth** [comment]
Working on getting this finished this morning.
